dspace/config/dspace.cfg

```diff
@@ -102,6 +102,13 @@ db.maxidle = -1
 # pool.
 #db.jndi = jdbc/dspace
 
+# Whether or not to allow for an entire 'clean' of the DSpace database.
+# Enabling it would allow your database owner to destroy all DSpace data, tables, etc
+# by simply running 'dspace database clean' from commandline.
+# WARNING: NEVER ENABLE IN PRODUCTION. This tool is for development/testing only.
+# (Defaults to 'true', i.e. disabled)
+# db.cleanDisabled = true
```

Comment:
This *is* a little unclear:  enabling *cleaning* lets you empty the database, but you enable cleaning by *dis*abling db.cleanDisabled (setting it false).  From reading the comment I was not entirely clear on whether the antecedent of "it" is cleaning or the configuration setting.